pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -459,6 +459,8 @@ message GenerativeSplitAndScatterSpec {
   optional bool exclusive_file_span_comparison = 22 [(gogoproto.nullable) = false];
   // MaxFileCount is the max number of files in an extending restore span entry.
   optional int64 max_file_count = 23[(gogoproto.nullable) = false];
+  // SQLInstanceIDs is a slice of SQL instance IDs available for dist restore.
+  repeated int32 sql_instance_ids = 24[(gogoproto.nullable) = false, (gogoproto.customname) = "SQLInstanceIDs"];
```

Comment:
Hi! The `(gogoproto.nullable) = false` here makes no sense, since this is a slice. I can see a warning about it when regenerating protos. Can you remove it?

Reply:
oof, we missed this, sorry about that. Is it worth creating a linter for this?

Reply:
Hey sorry I meant to clean this up earlier, I put out a PR to remove it: https://github.com/cockroachdb/cockroach/pull/129254

Reply:
Thanks!!